<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RahulMishra-PC" name="Default test(failed)" tests="1" failures="1" timestamp="31 May 2017 03:21:30 GMT" time="32.248" errors="0">
  <testcase name="@AfterTest close" time="0.0" classname="Test.TestLogin">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Test.TestLogin.close(TestLogin.java:58)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest close -->
  <testcase name="run" time="0.005" classname="Test.TestLogin">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at source.ReusableFunctions.captureScreenShot(ReusableFunctions.java:85)
	at Test.TestLogin.run(TestLogin.java:48)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- run -->
</testsuite> <!-- Default test(failed) -->
